templates/chat.html

```diff
@@ -0,0 +1,44 @@
+{% set active_page = "Chat" %}
+{% set page_title = "Chat" %}
+{% extends "base.html" %}
+
+{% block body %}
+<div class="slp">
+  <iframe id="frame" width="100%" src="http://www.google.com"> </iframe>
```

Comment:
noob question -- do we need to set `src` to `googe.com` here? It seems that this will be overwritten in Line 39-40.